### DISCUSSION TITLE
How to run database maintenance utilities in cnpg

### DISCUSSION THREAD

**Discussion by eshikhov** [cloudnative-pg/cloudnative-pg#1203] [Q&A]
Hi!

How to properly run database maintenance utilities such as vacuumlo, reindexdb, and others on a schedule with your operator?

**Comment by sxd** [comment]
Hi @eshikhov 

You can always use the native CronJob object from Kubernetes to connect to the database and run these kind of tools, do you think that there should be something implemented? specially because these kind of tools may slowdown your database, it's a tricky action to do not so easy to decide how to implement it

Regards,
